draft-ietf-tls-tls13.md

```diff
@@ -4147,7 +4186,7 @@ MUST meet the following requirements:
  * If not containing a "pre_shared_key" extension, it MUST contain both
    a "signature_algorithms" extension and a "supported_groups" extension.
  * If containing a "supported_groups" extension, it MUST also contain a
-   "key_share" extension, and vice versa. (an empty KeyShare.client_shares
+   "key_share" extension, and vice versa. (An empty KeyShare.client_shares
    vector is permitted)
```

Comment:
nit: if capitalizing "an", then this parenthetical sentence should also end with a period